.github/workflows/ci_ubuntu.yaml

```diff
@@ -85,6 +86,7 @@ jobs:
             -B build \
             -G "Ninja" \
             -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+            -DCMAKE_CXX_FLAGS="-Werror" \
```

Comment:
Why is this line different from the line 71?

Reply:
There are some unsuppressible/insolvable warnings in Qt5.